tests/lib/rules/no-this-render.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-this-render');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const TEST_FILE_NAME = 'some-test.js';
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-this-render', rule, {
+  valid: [
+    {
+      filename: TEST_FILE_NAME,
+      code: "run(() => { console.log('Hello World.'); });",
+    },
+    {
+      filename: TEST_FILE_NAME,
+      code: 'myCustomClass.render(myFunction);',
+    },
+    {
+      filename: TEST_FILE_NAME,
+      code: 'render.otherFunction(myFunction);',
+    },
```

Comment:
Can you add a valid test case for (just to ensure we avoid false positives / crashes):
```js
{
  filename: TEST_FILE_NAME,
  code: 'this.render.otherFunction()',
},

Reply:
Ping on adding test case.

Reply:
Sorry, I literally just submitted this, can you please take another look?